src/draw/lv_draw_image.h

```diff
@@ -71,7 +71,7 @@ struct _lv_draw_image_dsc_t {
     /**Describes how to blend the pixels of the image to the background.
      * See `lv_blend_mode_t` for more details.
      */
-    lv_blend_mode_t blend_mode : 3;
+    uint16_t blend_mode : 3;
```

Comment:
I'd rather go with this to keep the type for easier debugging.
```suggestion
    uint16_t blend_mode : 4;
```

Reply:
I have followed your suggestions.

Kenji Mouri